ls1intum/Artemis
# [Bugfix] Code Editor Tutor Assessment Inline Feedback fix points

### Checklist

### Motivation and Context



Currently, the points can not be used efficiently when the user wants to give inline feedback during programming exercise assessment. I just changed the column size for points and move the delete button into the bottom area for saving some space.


### Steps for Testing


1. Log in to Artemis
2. Create/Find a programming exercise
3. Set Manual Review and Enable online code editor
4. Participate to the exercise
5. As an instructor, start to assess this submission after due date
6. give an inline feedback on code editor
7. you should be able to see points properly
8. after save, click edit for this inline feedback
9. delete button should be placed at the bottom row

### Test Coverage



nothing's changed

### Screenshots



**Before:** 

<img src="https://user-images.githubusercontent.com/23650630/123345018-6cd88e00-d555-11eb-9f94-3fed2ba26fa2.png" width="500" />
<img src="https://user-images.githubusercontent.com/23650630/123345038-79f57d00-d555-11eb-9cac-b2907c5ec16c.png" width="500" />

**After:**
<img src="https://user-images.githubusercontent.com/23650630/123349940-3f421380-d55a-11eb-8133-67b9094cab59.png" width="500" />
<img src="https://user-images.githubusercontent.com/23650630/123349924-35b8ab80-d55a-11eb-807c-1d947f295583.png" width="500" />
